### PR TITLE
extend server42Spec

### PR DESCRIPTION
**Pull request by andriidovhan**

### CODE REVIEW

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/server42/Server42RttSpec.groovy [hunk 1]
```diff
@@ -279,6 +332,200 @@ class Server42RttSpec extends HealthCheckSpecification {
...
+                        !switchIdsConnectedToS42.contains(secondSwP.src.dpId)
+            }
+            firstOk && fl2SwPair
+        }
+        assumeTrue("Required switch pairs were not found in the given topology",
+                fl1SwPair.asBoolean() && fl2SwPair.asBoolean())
+
+        and: "server42 is enabled on the src sw of the first switch pair"
+        def flowRttFeatureStartState = changeFlowRttToggle(true)
+        def initialSwitchRtt = [].collectEntries { [fl1SwPair.src, changeFlowRttSwitch(fl1SwPair.src, true)] }
```
Comment (rtretyak) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/server42/Server42RttSpec.groovy:443]: I'm not sure what this line is supposed to do, but right now it is incorrect, you are trying to iterate over an empty array

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/server42/Server42RttSpec.groovy [hunk 2]
```diff
@@ -279,6 +332,200 @@ class Server42RttSpec extends HealthCheckSpecification {
...
+        def involvedSwitches = [fl1SwPair.src, fl1SwPair.dst, fl2SwPair.src, fl2SwPair.dst]*.dpId.unique()
+        Wrappers.wait(RULES_INSTALLATION_TIME) {
+            involvedSwitches.each { swId ->
+                northbound.validateSwitch(swId).verifyRuleSectionsAreEmpty(["missing", "excess", "misconfigured"])
+            }
+        }
+        def switchesAreValid = true
+
+        and: "server42 stats are available for the flow2 in the forward direction"
+        TimeUnit.SECONDS.sleep(STATS_FROM_SERVER42_LOGGING_TIMEOUT + WAIT_OFFSET)
```
Comment (rtretyak) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/server42/Server42RttSpec.groovy:502]: This should be reworked, we can't afford that long sleep. `STATS_FROM_SERVER42_LOGGING_TIMEOUT` is just a random number that was decided to be used as timeout, it has no real application and you should not rely on it here

Comment (andriidovhan) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/server42/Server42RttSpec.groovy:563]: added `todo` and `low_priority` tag.
Actually don't have any idea now how to rework.

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/server42/Server42RttSpec.groovy (hunk 1 above)
Comment (andriidovhan) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/server42/Server42RttSpec.groovy]: ops, fixed.
thx.

**Review by rtretyak — APPROVED** [review]

**Pull request merged**